modules/apps/27-interchain-accounts/module.go

```diff
@@ -46,6 +49,12 @@ func (AppModuleBasic) Name() string {
 	return types.ModuleName
 }
 
+// IsOnePerModuleType implements the depinject.OnePerModuleType interface.
+func (AppModule) IsOnePerModuleType() {}
```

Comment:
any idea what does this API infer to the depinject/runtime setup?

Reply:
https://docs.cosmos.network/main/architecture/adr-057-app-wiring#dependency-injection touches on it *slightly*, haven't had time to read further into it tho